OpenDirectoryDownloader/DirectoryParser.cs

```diff
@@ -2066,9 +2066,9 @@ private static HeaderInfo GetHeaderInfo(IElement header)
 
             headerName = headerName.ToLower();
 
-            headerName = Regex.Replace(headerName, @"[^a-zA-Z0-9\s一-龥äöüÄÖÜß]", string.Empty);
+            headerName = Regex.Replace(headerName, @"[^\u00BF-\u1FFF\u2C00-\uD7FF\w]", string.Empty);
```

Comment:
Do you have more info about this other regex?

Reply:
The problem with the old RegEx essentially was that the ranges you specified (A-z,a-z, etc.) didn't include some japanese characters (chinese Hanzi seemed to work though). The new RegEx uses unicode ranges instead, in order to include most every forgein characters.

With the old pattern, some parts of the column headings were ignored and the type detection didn't work.

I can link the thread where I got it from once I get to a computer :)

Reply:
OK, np, I'll see it later.

Reply:
see the second answer here: https://stackoverflow.com/questions/3009993/what-would-be-regex-for-matching-foreign-characters

Reply:
OK, thanks, good 👍